azure-iot-provisioning-devicesdk/tests/security/test_sk_security_client.py

```diff
@@ -0,0 +1,35 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import pytest
+from azure.iot.provisioning.devicesdk.security.sk_security_client import SymmetricKeySecurityClient
+
+fake_symmetric_key = "Zm9vYmFy"
+key_name = "registration"
+fake_provisioning_host = "beauxbatons.academy-net"
+fake_registration_id = "MyPensieve"
+module_id = "Divination"
+fake_id_scope = "Enchanted0000Ceiling7898"
+signature = "IsolemnlySwearThatIamuUptoNogood"
+expiry = "1539043658"
+
+
+def test_getters_after_create_security_client():
```

Comment:
Test that the properties are not settable as well

Reply:
are u referring to renaming ?

---
In reply to: [270213951](https://github.com/Azure/azure-iot-sdk-python-preview/pull/59#discussion_r270213951) [](ancestors = 270213951)

Reply:
No I'm referring to you need to add another test to test that the properties are not settable.

---
In reply to: [271106251](https://github.com/Azure/azure-iot-sdk-python-preview/pull/59#discussion_r271106251) [](ancestors = 271106251,270213951)